src/Data/Connection/MenuItemConnectionResolver.php

```diff
@@ -50,10 +50,9 @@ public function get_query_args() {
 			$query_args['meta_value'] = (int) $this->args['where']['parentDatabaseId']; // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_value
 		}
 
-		if ( ! empty( $this->args['where']['parentId'] ) ) {
-
-				$query_args['meta_key']   = '_menu_item_menu_item_parent'; // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key
-				$query_args['meta_value'] = $this->args['where']['parentId']; // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_value
+		if ( isset( $this->args['where']['parentId'] ) ) {
```

Comment:
I think this conditional needs to be more explicit, since `$this->get_args()` runs before `get_query_args()`, and a bad (vs falsy) input value will cause `Utils::get_database_id_from_id()` to return `false`

Reply:
ok, so probably true with the `isset` in place a few lines above as well?

Reply:
No because `parentDatabaseId` doesn't get passed through that Utils function, so `isset()` there is fine.

I think `( 0 === $this->args[...] || ! empty( $this->args[...] ) )` logic we want.

Reply:
ok, I can run with that.

Reply:
> I think ( 0 === $this->args[...] || ! empty( $this->args[...] ) ) logic we want.

We need to check `isset( $this->args['where']['parentId'] )` before evaluating if it's equal to zero. 

So it'll need to be something like `! empty( $... ) || ( isset( $... ) && 0 === $... )`

Reply:
see: https://github.com/wp-graphql/wp-graphql/pull/2532/commits/6bac3841342bf36413f979721bfe8f2291301030